nautobot/extras/forms/forms.py

```diff
@@ -2070,6 +2071,23 @@ class Meta:
 #
 # Webhooks
 #
+class WebhookBulkEditForm(NautobotBulkEditForm):
+    """Bulk edit form for Webhook objects."""
+
+    pk = forms.ModelMultipleChoiceField(queryset=Webhook.objects.all(), widget=forms.MultipleHiddenInput())
+    enabled = forms.BooleanField(required=False)
+    type_create = forms.BooleanField(required=False)
+    type_update = forms.BooleanField(required=False)
+    type_delete = forms.BooleanField(required=False)
+
```

Comment:
I'd think that any of the following might be useful:

- add/remove content_types
- payload_url, secret, ca_file_path